keps/sig-node/3675-resource-plugin-manager/README.md

```diff
@@ -0,0 +1,1002 @@
+Additionally, available resources will be updated.
+
+
+###### How can someone using this feature know that it is working for their instance?
+
+- [ ] Events
+  - Event Reason: 
+- [x] API .status
+  - Condition name: CCI Driver readiness and CCI Status field can be introduced
+  - Other field: 
+- [ ] Other (treat as last resort)
+  - Details:
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+Pods not using CCI Drivers continue to work as before. Consistent state handling for 
+pods which require a CCI Driver.
+
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+<!--
+Pick one more of these and delete the rest.
+-->
+TBD in Beta
+- [ ] Metrics
+  - Metric name:
+  - [Optional] Aggregation method:
+  - Components exposing the metric:
+- [ ] Other (treat as last resort)
+  - Details:
+
+###### Are there any missing metrics that would be useful to have to improve observability of this feature?
+
+<!--
+Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
+implementation difficulties, etc.).
+-->
+TBD in Beta
+
+### Dependencies
+
+None
+<!--
+This section must be completed when targeting beta to a release.
+-->
+
+###### Does this feature depend on any specific services running in the cluster?
+
+A CCI Driver (daemonset) will be required to handle pods which need CCI Driver resource management.
+
+### Scalability
+
+The CCI Resource Manager approach resembles device manager protocol. The scalability and performance impact will be similar to the case of handling device plugins.
+
+Further performance benchmarks will be done in Beta Phase
+
+
+###### Will enabling / using this feature result in any new API calls?
+
+For Pods requiring CCI Driver the admission, container add and container removal operations will be handled via 
+api call to external CCI Driver. Remaining pods should be handled as before.
+
+###### Will enabling / using this feature result in introducing new API types?
+
+A helper API to create CCI Drivers (CCI Driver Factory) can be integrated as staged repo.
+
+###### Will enabling / using this feature result in any new calls to the cloud provider?
+
+No 
+
+###### Will enabling / using this feature result in increasing size or count of the existing API objects?
+
+This will not be true for Alpha.  For Beta, where there may be multiple drivers, it may be prudent to consider this.
+
+###### Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?
+
+No
+
+###### Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?
+
+No  
+
+### Troubleshooting
+  
+In alpha phase we propose to handle bootstrapping and failures due unavailable drivers 
+by going back to best-effort QoS. In post-alpha phase we will consider solutions of the bootstrapping and driver failure problem
```

Comment:
Can you elaborate what you mean by "going back to best-effort QoS"?

Reply:
done